src/style.css

```
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: italic;
+  font-weight: normal;
+}
+
+body {
+  box-sizing: border-box;
+}

```

you can add this style for all elements with *
```suggestion
* {
  box-sizing: border-box;
}

```